banzai/main.py

```diff
@@ -288,9 +288,9 @@ def update_db():
         logger.error('Could not populate instruments table: {error}'.format(error=logs.format_exception()))
 
 
-def add_bpm():
-    parser = argparse.ArgumentParser(description="Add a bad pixel mask to the db.")
-    parser.add_argument('filepath', help='Full path to Bad Pixel Mask file')
+def add_cal():
```

Comment:
Nitpick, but we could call this `add_super_calibration` - since that a bit more descriptive about what it actually does.

We're still working on updating our nomenclature in the pipeline to get rid of the term "master," so "super" is our new replacement.